src/modules/template-details/TemplateDetails.tsx

```diff
@@ -11,6 +11,8 @@ import ShareIcon from '@material-ui/icons/Share';
 
 import { Button, Loader, More } from 'ui';
 
+import { useAuthProvider } from 'core/auth';
```

Comment:
Put this After utils import - we have convention when put imports based by from length and separate them by modules with enter